index.js

```diff
@@ -156,7 +163,7 @@ AFRAME.registerComponent('loader-3dtiles', {
     }
   },
   tick: function (t, dt) {
-    if (this.runtime) {
+    if (this.runtime && this.viewportSize) {
```

Comment:
Or move `this.viewportSize` creation before the `await this._nextFrame();` in `init`?

Reply:
yes this makes more sense to fix in init once per session per Vincent's suggested instead of checking this in every tick, it is a better pattern. could you try that @Algorush and see if it still works the same? if it no longer works expected with that change then it's fine to revert back to what you have here instead.

Reply:
yes, Vincent's suggestion works good. I added new commit. But now I found an error when switching to ortho camera. And I still don’t understand why...